odoo/addons/test_orm/tests/test_properties.py

```diff
@@ -2039,25 +2036,27 @@ def test_properties_field_search_many2many(self):
         }]
         self.message_2.attributes = {'mymany2many': [partners[1].id]}
         self.message_3.attributes = {'mymany2many': [partners[2].id]}
+
+        # NOTE: filtered won't work because each message can point to a
+        # different model or even have a different data type
```

Comment:
I though that was going to be forbidden.  To be checked.

Reply:
The note is here just to explicitly say that this is not supported. The code is untested for this case.